specification.rst

```diff
@@ -2578,33 +2587,43 @@ entry-encyclopedia
     An entry in an encyclopedia or similar reference work
 
 figure
-    a
+    A figure illustrating or presenting research data; may be in any format 
```

Comment:
1. "figure" is circular; no"; maybe "graphic"?
2. perhaps should strike "research"? that qualifier seems to make it too narrow?

Reply:
The major question is what is the difference between `figure` and `graphic`?

Reply:
Thinking this through

In terms of non-written material, we have three broad item types:
  1. `graphic`: still visual items
  2. `song`: audio items
  3. `motion_picture`: video items

Those cover the major citation needs, with `genre` and `medium` providing subtypes and format descriptions. 

But then there's `figure`. What exactly is that for? For example, a biology paper might contain a still image of a protein. That could be a `graphic`. A video demonstrating how the protein folds could be a `motion_picture`. But both of those could also be a `figure`. I think there is potential value in having a specific item type to refer to things like [this figure extracted from an article](https://figshare.com/articles/Disruption_of_lipid_rafts_reduces_I_sub_Na_sub_and_alters_the_impact_of_twin_NEPs_on_the_inward_current_/12455345) or [this figure we posted standalone](https://psyarxiv.com/bm28v/). I'm not too worried about their being conceptual overlap here. If an item can reasonably be described as a `figure` for research, then the type is appropriate?

Perhaps the description for figure should be:
`An illustration or representation of data;`
`May be in any format`

We could potentially say that `figure` specifically is a component extracted from a journal article, preprint, etc. (e.g., many medical journals assign individual DOIs to each figure and table so that they can be separately linked to and archived). I don't think we need to be that restrictive (e.g., the second item I linked to above).

Reply:
I was just about to ask whether we need a figure type at all. Since I'd never cite the figure, just always the article that contains a figure. But in your last paragraph you say: 

> We could potentially say that figure specifically is a component extracted from a journal article, preprint, etc. (e.g., many medical journals assign individual DOIs to each figure and table so that they can be separately linked to and archived). I don't think we need to be that restrictive (e.g., the second item I linked to above).

Maybe that warrants for a table item type?

Reply:
One argument for `figure` existing is that sources like CrossRef and figshare have `figure` type and serve that as the item type. Assigning these to `figure` avoids the ambiguity of whether they should be `graphic`, `motion_picture`, etc. 

I'd say a table could go under `figure` if anything (that seems to be what CrossRef does).